examples/commands/play.js

```diff
@@ -70,6 +70,11 @@ module.exports = {
         }
 
         const first = collected.first().content;
+        
+        if(isNaN(first)) {
+          if (!player.queue.current) return player.destroy();
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585534)
```suggestion
          if (!player.queue.current) {return player.destroy();}
```